core/storage/base_model/gae_models.py

```diff
@@ -725,7 +725,7 @@ def create(
             committer_id: str,
             commit_type: str,
             commit_message: Optional[str],
-            commit_cmds: Union[Dict[str, Any], List[Dict[str, Any]], None],
+            commit_cmds: AllowedCommitCmdsListType,
```

Comment:
This change (from Union[Dict[str, Any], List[Dict[str, Any]], None] to AllowedCommitCmdsListType) is great. Can you make similar changes to other places in this PR where the type is similarly messy?

Reply:
Done, wherever it is applicable!